src/index.js

```diff
@@ -23,7 +23,7 @@ window.onload = () => {
       <hr>`;
   });
 
-  const changeTaskStatus = (index, status) => {
+  function changeTaskStatus(index, status) {
     todos.filter((todo, todoIndex) => {
       if (index === todoIndex) {
         todo.completed = status;
```

Comment:
Great job @1GORDON 

- [ ] Kindly use ES6 syntax for all your functions and methods as stated in [javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) across all your javascript files.